pkg/lib/authenticationflow/declarative/data_account_linking.go

```diff
@@ -0,0 +1,43 @@
+package declarative
+
+import (
+	authflow "github.com/authgear/authgear-server/pkg/lib/authenticationflow"
+	"github.com/authgear/authgear-server/pkg/lib/authn/identity"
+	"github.com/authgear/authgear-server/pkg/lib/config"
+	"github.com/authgear/authgear-server/pkg/util/slice"
+)
+
+type AccountLinkingIdentificationOption struct {
+	Identifcation config.AuthenticationFlowIdentification `json:"identification"`
+
+	// MaskedDisplayName is specific to login id
+	MaskedDisplayName string `json:"masked_display_name"`
```

Comment:
Not sure if we need to have a display name for oauth identity.
If yes, we need to define a logic for it.

Reply:
As discussed, display email -> username -> phone

Reply:
After a second thought, it seems username should be displayed first for oauth provider, because username must be specific to that provider, while email / phone can be used in other place.